docs/reference/connectors.md

```diff
@@ -0,0 +1,123 @@
+# Connectors
+
+This resource pack supports the following connection configurations:
+
+* Environment Variables
+* Credentials File
+* Managed Service Identity (MSI) Connector
+
+For any of these connection strategies you'll need a Subscription ID and Tenant
+ID. These values may be found in the Azure Portal. This guide assumes you are
+using a Service Principal. See the
```

Comment:
This is a poor name that doesn't communicate anything when used in context. We should work around that by stating that a service principal is a particular designation for an account or set of credentials or whatever it is. (See? I'm still not clear on it!)

Reply:
Service principal is a thing in the Azure docs: https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal. What would make this more clear?